x-pack/plugins/profiling/server/lib/setup/get_setup_instructions.ts

```diff
@@ -34,11 +34,16 @@ export async function getSetupInstructions({
 }): Promise<SetupDataCollectionInstructions> {
   const profilerAgent = await fetchFindLatestPackageOrThrow('profiler_agent', { prerelease: true });
   const collectorPolicy = await getCollectorPolicy({ packagePolicyClient, soClient });
+  const symbolizerPolicy = await getSymbolizerPolicy({ packagePolicyClient, soClient });
```

Comment:
Do we need to use this value later? Or is this a check to make sure the symbolizer policy exists?

Reply:
It makes sure the symbolizer policy exists.